src/test/java/com/google/api/codegen/testdata/go_library.baseline

```diff
@@ -621,6 +621,181 @@ func (iterator *StringIterator) Next() (string, error) {
     iterator.currentIndex++
     return result, nil
 }
+============== file: library_service/client_test.go ==============
+package library_service
+
+import (
+    gax "github.com/googleapis/gax-golang"
+    "golang.org/x/net/context"
+    google_example_library_v1 "google.golang.org/cloud/library/proto/google/example/library/v1"
+)
+
+func ExampleNewClient() {
+    ctx := context.Background()
+    opts := []gax.ClientOption{ /* Optional client parameters. */ }
+    client, err := NewClient(ctx, opts...)
+    _, _ = client, err
+}
+
+func ExampleClient_CreateShelf() {
+    ctx := context.Background()
+    client, _ := NewClient(ctx)
```

Comment:
Users can't copy this code because they will have to actually write `library_service.NewClient`. Consider repackaging the test file into `library_service_test` then import `library_service`.